src/modules/aws.rs

```diff
@@ -68,8 +68,8 @@ fn get_aws_profile_and_region(context: &Context) -> (Option<Profile>, Option<Reg
         .iter()
         .find_map(|env_var| context.get_env(env_var));
     let region = context
```

Comment:
Not a blocker but maybe it makes sense to refactor this so it is the same as how we are finding the profile, something like:

```rust
 ​    ​let​ refion_env_vars ​=​ ​vec!​[​​"AWS_REGION", AWS_DEFAULT_REGION"]; 
 ​    ​let​ region ​=​ region_env_vars 
 ​        .​iter​() 
 ​        .​find_map​(​|​env_var​|​ context.​get_env​(env_var));
```